updating/updating_a_cluster/updating_disconnected_cluster/disconnected-update-osus.adoc

```diff
@@ -135,6 +135,13 @@ The release image signature config map allows the Cluster Version Operator (CVO)
 ====
 * The current release and update target release images are mirrored to a locally accessible registry.
 * A recent graph data container image has been mirrored to your local registry.
+* You have the most recent version of the OSUS Operator.
++
+[NOTE]
+====
+If it has been some time since you installed or last updated the OSUS Operator, you may need to update the Operator to its latest version.
```

Comment:
I can soften the end of this sentence to "there might be a newer version of the OSUS Operator available" if always having the latest version isn't something we want to present as a hard requirement. In that case, I could also rephrase line 138 above.